src/Sulu/Bundle/AdminBundle/Resources/js/components/Popover/README.md

```diff
@@ -0,0 +1,42 @@
+The Popover component is a simple element which can contain other kinds of components. A good usecase example 
```

Comment:
It doesn't say what the component is doing at all 😉 It should mention that it is for showing content above all other elements, and that it can be positioned by using an anchor element.